src/kernel/arch/x86/arch.zig

```diff
@@ -45,3 +57,85 @@ pub fn inb(port: u16) u8 {
 pub fn ioWait() void {
     outb(0x80, 0);
 }
+
+// Can't use gdt.KERNEL_DATA_OFFSET in the assembly bellow, so have to store in a constant.
+const KERNEL_DATA_OFFSET: u16 = gdt.KERNEL_DATA_OFFSET;
```

Comment:
You may be able to add `.extern KERNEL_DATA_OFFSET` to the inline assembly and use it in there, much like I did in the [boot code](https://github.com/SamTebbs33/pluto/blob/e90d5e4c87ecb14da608bed40c98ee7892f88be6/src/kernel/arch/x86/boot.zig#L70), not sure though

Reply:
This will mean i will have to export the KERNEL_DATA_OFFSET, which i don't want, but found a different way, u'll see in the next commit